### DISCUSSION TITLE
Issue at Startup onboarding guide.

### DISCUSSION THREAD

**Discussion by AndriiTsok** [community/community#30230] [New to GitHub]
We are experiencing an issue with the Startup onboarding guide. One of the steps **cannot be completed**. 

Despite changing the base permission the **Customize members' permissions** does not get completed.

Organization: https://github.com/TRYON-Technology

<img width="950" alt="Screen Shot 2022-08-20 at 2 31 15 PM" src="https://user-images.githubusercontent.com/733224/185761469-946de370-9ef2-4f02-91bd-f1a1229b50f0.png">
.

**Comment by queenofcorgis** [comment]
Hi @AndriiTsok 👋🏼 

Are you still experiencing this?

**Reply by AndriiTsok** [reply]
Hi @queenofcorgis we managed to resolve it. But we still can't remove the tasks from our private org page. They all are completed now and cannot disable/hide them.

**Reply by shayded-exe** [reply]
I'm experiencing this. Tried changing permissions but it won't mark it completed.

**Reply by Pouya-moh** [reply]
We have github Teams and have the same issue. Tried both to elevate and demote base permission and the card still stays there.

**Reply by tedivm** [reply]
I'm also dealing with this, and it's really frustrating. Github should have a way to dismiss this garbage.

**Comment by code-irisnk** [comment]
Same here with @elrant

**Reply by AlejandroVindel** [reply]
Same in my GitHub Organization :(

**Comment by tisonkun** [comment]
I cannot check "Discover new GitHub features" so the onboarding page is always there.

I don't want to see that page, though :(

**Reply by tisonkun** [reply]
<img width="928" alt="image" src="https://github.com/community/community/assets/18818196/5c3c3f62-9e3a-4fdf-855a-91c022f6bee3">

**Reply by tisonkun** [reply]
cc @queenofcorgis

**Reply by tisonkun** [reply]
Well. I found "You can  on this page and bring them back later." button now ..

**Comment by jozefizso** [comment]
How can I disable the onborading screen? I cannot check the individual cards, and even when the repositories does have CI/CD setup, the onboarding still nags about this.

**Comment by dniccum** [comment]
@jozefizso @tisonkun  There is a link tucked up on the right side of your sidebar. It's small, but it should do the trick:

![CleanShot 2024-07-25 at 08 19 25](https://github.com/user-attachments/assets/429e919e-7c68-4899-b6b6-ac56660b5423)
